contracts/AcceleratingDistributor.sol

```diff
@@ -260,6 +249,24 @@ contract AcceleratingDistributor is ReentrancyGuard, Ownable, Multicall {
         emit Exit(stakedToken, msg.sender, stakingTokens[stakedToken].cumulativeStaked);
     }
 
+    /**************************************
+     *           PUBLIC FUNCTIONS         *
+     **************************************/
+
+    /**
+     * @notice Recover an ERC20 token either dropped on the contract or excess after the staking program ends.
+     * @dev Any wallet can call this function as it will only ever send tokens to the owner of the distributor.
+     * @param tokenAddress The address of the token to recover.
+     * @param amount The amount of the token to recover.
+     */
+    function recoverErc20(address tokenAddress, uint256 amount) external nonReentrant {
```

Comment:
this will be removed in favor of the method added here: https://github.com/across-protocol/across-token/pull/8